edit words (#246)
jaybee84/ml-in-rd: content/03.combining-datasets.md

In the first approach, appropriate training, evaluation, and held-out validation sets could be constructed by combining multiple rare disease cohorts (Figure [@fig:1]a, Box 2).
When combining datasets, special attention should be directed towards data harmonization since data collection methods can differ between cohorts.
Without careful selection of aggregation methods, one may introduce variability into the combined dataset and negatively impact the ML model’s ability to learn or detect meaningful signals.
Steps like reprocessing data using a single pipeline, using batch correction methods [@doi:10.1093/biostatistics/kxj037; @doi:10.1093/nar/gku864], and normalizing raw values appropriately without affecting the underlying variance in the data [@doi:10.1186/gb-2010-11-3-r25, @doi:10.1371/journal.pcbi.1003531, @doi:10.1186/s13059-014-0550-8] may be necessary to mitigate unwanted variability. (Figure [@fig:1]a)
Data harmonization may also entail standardization of sample labels using biomedical ontologies to normalize how samples are annotated across multiple datasets.

> Ways to improve the quality of a dataset include improving the accuracy of metadata (both the description of technical variables and biologically relevant phenotypes) for each sample in the dataset.

Reviewer: ```suggestion
Improving the accuracy of metadata (both the description of technical variables and biologically relevant phenotypes) for each sample in the dataset can improve that dataset's quality.
```
